[2023-09-14T01:04:15.719+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test01.S3Write manual__2023-09-14T01:04:14.369863+00:00 [queued]>
[2023-09-14T01:04:15.724+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test01.S3Write manual__2023-09-14T01:04:14.369863+00:00 [queued]>
[2023-09-14T01:04:15.724+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-09-14T01:04:15.731+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): S3Write> on 2023-09-14 01:04:14.369863+00:00
[2023-09-14T01:04:15.741+0000] {standard_task_runner.py:57} INFO - Started process 113 to run task
[2023-09-14T01:04:15.746+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'test01', 'S3Write', 'manual__2023-09-14T01:04:14.369863+00:00', '--job-id', '110', '--raw', '--subdir', 'DAGS_FOLDER/OrdersRetention.py', '--cfg-path', '/tmp/tmp4s8dww0t']
[2023-09-14T01:04:15.750+0000] {standard_task_runner.py:85} INFO - Job 110: Subtask S3Write
[2023-09-14T01:04:15.794+0000] {task_command.py:415} INFO - Running <TaskInstance: test01.S3Write manual__2023-09-14T01:04:14.369863+00:00 [running]> on host 2205d3bdd974
[2023-09-14T01:04:15.850+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test01' AIRFLOW_CTX_TASK_ID='S3Write' AIRFLOW_CTX_EXECUTION_DATE='2023-09-14T01:04:14.369863+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-14T01:04:14.369863+00:00'
[2023-09-14T01:04:17.884+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/google/cloud/bigquery/table.py:2155: UserWarning: A progress bar was requested, but there was an error loading the tqdm library. Please install tqdm to use the progress bar functionality.
  record_batch = self.to_arrow(

[2023-09-14T01:04:36.091+0000] {logging_mixin.py:151} INFO -                              _id  ...                     owner
0              ZZdPb6nW37r56WNMH  ...                       NaN
1              3BCQ9vtMmQK4bevqZ  ...                       NaN
2              FTA3zkcJbcDrNqCXi  ...                       NaN
3              nGbi3HLWaRf9gNc5y  ...         K3NnKA4FHHJ5JHcQX
4              jWWfumEkoYmhA6JtJ  ...         tmy8ATd6P5S3QtBhH
...                          ...  ...                       ...
471292  6311c7edb0cf2d6bff26e8f7  ...  62be77603255fc0fcdf67d17
471297  6315a7f725d0c332b49777f1  ...  5f322ead79e6ab678b6b72fe
471302  631823aaf406909f910ff88f  ...  5f322ead79e6ab678b6b72fe
471303  631823eff406909f910ffc85  ...  5f322ead79e6ab678b6b72fe
471308  6318316e4fb7fe86b3356bb2  ...  63182ad54fb7fed85235259f

[467826 rows x 6 columns]
[2023-09-14T01:04:36.269+0000] {logging_mixin.py:151} INFO - s3 bucket connected!
[2023-09-14T01:04:38.968+0000] {logging_mixin.py:151} INFO - 데이터가 S3 버킷 wingeatdata의 practice/2022-9-7-5.csv 위치에 저장되었습니다.
[2023-09-14T01:04:38.969+0000] {logging_mixin.py:151} INFO - ----------------------------------------------------------2022-09-07 05:51:42.002000
[2023-09-14T01:04:38.995+0000] {python.py:194} INFO - Done. Returned value was: 2022-09-07 05:51:42.002000
[2023-09-14T01:04:39.016+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=test01, task_id=S3Write, execution_date=20230914T010414, start_date=20230914T010415, end_date=20230914T010439
[2023-09-14T01:04:39.108+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-14T01:04:39.124+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
